packages/ra-ui-materialui/src/field/FileField.tsx

```diff
@@ -67,7 +67,7 @@ export const FileField = <
                     const srcValue = get(file, src) || title;
 
                     return (
-                        <li key={index}>
+                        <li key={index} onClick={e => e.stopPropagation()}>
```

Comment:
Why did you put the event handler on the `li` rather than the `Link` element?

Reply:
Indeed putting it on the `Link` element works just as well and makes more sense, nice catch.